eng/pipelines/coreclr/templates/install-diagnostic-certs.yml

```diff
@@ -14,12 +14,23 @@ steps:
       SecretsFilter: ${{ join(',', parameters.certNames) }}
     displayName: 'Download secrets: Diagnostic Certificates'
 
-  - task: EsrpClientTool@2
-    displayName: Download ESRPClient
+  - task: NuGetCommand@2
+    displayName: Install ESRPClient Tool
+    inputs:
+      command: 'custom'
+      arguments: 'install microsoft.esrpclient
+                -Source https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
+                -DependencyVersion Highest -OutputDirectory $(Build.StagingDirectory)'
```

Comment:
Any chance you can ask the ESRP folks if they'll update the EsrpClientTool task to download a newer version?

Not blocking of course.

Reply:
I asked and they said two days 2 weeks ago... but it's still returning the old one.